packages/flutter_android_lifecycle/README.md

```diff
@@ -0,0 +1,41 @@
+# Flutter Android Lifecycle Plugin
+
+[![pub package](https://img.shields.io/pub/v/flutter_android_lifecycle.svg)](https://pub.dartlang.org/packages/flutter_android_lifecycle)
+
+A Flutter plugin for Android to allow other Flutter plugins to access an Android `Lifecycle` object
+in the plugin's binding.
+
+*Note*: This plugin is still under development, and some APIs might not be available yet.
+
+## Installation
+
+Add `flutter_android_lifecycle` as a [dependency in your pubspec.yaml file](https://flutter.io/using-packages/).
+
+## Example
+
+Use a `FlutterLifecycleAdapter` within another Flutter plugin's Android implementation, as shown
```

Comment:
I think it's worth explaining here why we want to do this instead of using the engine's lifecycle directly.

Reply:
done